packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -7309,7 +7340,7 @@ function lookupNetworkTests({
   operation: (controller: NetworkController) => Promise<void>;
 }) {
   describe('if the network ID and network details requests resolve successfully', () => {
-    const validNetworkIds = [12345, '12345', toHex(12345)];
+    const validNetworkIds = [12345, '12345', toHex(12345)]; // ?
```

Comment:
Some chains report their network IDs in hex format: https://github.com/MetaMask/metamask-extension/issues/19151